app/assets/javascripts/datetimepicker.js

```diff
@@ -0,0 +1,103 @@
+$(document).ready(function () {
+
```

Comment:
This should probably be `$(document).on("turbolinks:load",` and so on as we are using turbolinks.

Reply:
No, the turbolink event is already fired when the js is loaded dynamically (which is the case as we use `javascript_include_tag`).

> Any <script src="xxx"> label added to document dynamically will be loaded asynchronously,
and the 'turbolinks:load' event is always triggered before the script finish loading.
That's why 'turbolinks:load' event in the new script not work .
~ from [here](https://github.com/turbolinks/turbolinks/issues/326#issuecomment-339541823)

Instead, I fixed the issue by adding an additional event listener for the turbolink load event that gets fired when the page is visited again via back and forth navigation in the browser. We then look for stale datetimepickers and simply remove them from the DOM.

Reply:
Ah, that is good to know.